assets/swagger.json

```diff
@@ -4699,6 +4699,10 @@
       "type": "object",
       "title": "AppProjectSpec is the specification of an AppProject",
       "properties": {
+        "allDestinationsMustMatch": {
+          "description": "AllDestinationsMustMatch, if true, means that a resource's destination cluster and namespace must match all patterns in the `destinations` list to be allowed.\nIf AllDestinationsMustMatch is false, the resource's destination only needs to match one of the cluster/namespace patterns in the `destinations` list.",
```

Comment:
take care of \n

Reply:
You mean remove it?

Reply:
I think `\n` is fine here. This is the OpenAPI spec, and it's up to OpenAPI tooling to properly handle the newline.